packages/alice_dio/test/alice_dio_adapter_test.dart

```diff
@@ -0,0 +1,229 @@
+import 'package:alice/core/alice_core.dart';
+import 'package:alice/model/alice_http_call.dart';
+import 'package:alice/model/alice_http_request.dart';
+import 'package:alice/model/alice_http_response.dart';
+import 'package:alice_dio/alice_dio_adapter.dart';
+import 'package:dio/dio.dart';
+import 'package:mocktail/mocktail.dart';
+import 'package:test/test.dart';
+
+import 'alice_core_mock.dart';
+
+void main() {
+  late AliceCore aliceCore;
+  late AliceDioAdapter aliceDioAdapter;
+  late Dio dio;
+  setUp(() {
+    registerFallbackValue(AliceHttpCall(0));
+    registerFallbackValue(AliceHttpResponse());
+    aliceCore = AliceCoreMock();
+    when(() => aliceCore.addCall(any())).thenAnswer((_) => {});
+    when(() => aliceCore.addResponse(any(), any())).thenAnswer((_) => {});
+    aliceDioAdapter = AliceDioAdapter();
+    aliceDioAdapter.injectCore(aliceCore);
+
+    dio = Dio(BaseOptions(followRedirects: false))
+      ..interceptors.add(aliceDioAdapter);
+  });
+
+  group("AliceDioAdapter", () {
+    test("should handle GET call with json response", () async {
+      await dio.get<void>('https://httpbin.org/json',
```

Comment:
You'll need to mock this. Network calls are not allowed from a test. Check how we do it in Chopper https://github.com/lejard-h/chopper/blob/33372bf395977f20bf0184d8c3b9deb07656461a/chopper/test/client_test.dart#L24